packages/x-codemod/src/v6.0.0/view-components-rename/view-components-rename.test.ts

```diff
@@ -0,0 +1,44 @@
+import path from 'path';
+import { expect } from 'chai';
+import jscodeshift from 'jscodeshift';
+import transform from '.';
+import readFile from '../../util/readFile';
+
+function read(fileName) {
+  return readFile(path.join(__dirname, fileName));
+}
+
+const TEST_FILES = ['community-nested-imports', 'community-root-imports', 'pro-root-imports'];
+
+describe('v6.0.0', () => {
+  describe('view-components-rename', () => {
+    TEST_FILES.forEach((testFile) => {
+      const actualPath = `./actual-${testFile}.spec.tsx`;
+      const expectedPath = `./expected-${testFile}.spec.tsx`;
```

Comment:
Nitpick: I have specifically gone with using `js` files instead of `ts` for test fixtures in order to avoid red IDE and additional TS processing for files we know will usually have errors.
But if you think it adds value having them authored in `ts` as well—by no means, we can go with. 👌

Reply:
I am testing the renaming of the exported types, that's why I went for TS :+1:

Reply:
Ah, fair point 🤦